mtalg/random/__init__.py

```diff
@@ -1,19 +1,164 @@
 
-for method in get_methods(__RNG):
-    argsspec = inspect.getfullargspec(getattr(__RNG, method))
-    args_all, defaults = argsspec.args[1:], argsspec.defaults or []
-    kwargs = {k: v for k, v in zip(args_all[len(args_all) - len(defaults):], defaults)}
-    args = args_all[:len(args_all) - len(defaults)]
 
+def binomial(*args, **kwargs):
+    return _RNG.binomial(*args, **kwargs)
 
-    def func(*args, **kwargs):
-        getattr(__RNG, method)(*args, **kwargs)
 
+def chisquare(*args, **kwargs):
+    return _RNG.chisquare(*args, **kwargs)
 
-    exec(f'{method}=func')
+
+def exponential(*args, **kwargs):
+    return _RNG.exponential(*args, **kwargs)
+
+
+def f(*args, **kwargs):
+    return _RNG.f(*args, **kwargs)
+
+
+def gamma(*args, **kwargs):
+    return _RNG.gamma(*args, **kwargs)
+
+
+def geometric(*args, **kwargs):
+    return _RNG.geometric(*args, **kwargs)
+
+
+def gumbel(*args, **kwargs):
+    return _RNG.gumbel(*args, **kwargs)
+
+
+def hypergeometric(*args, **kwargs):
+    return _RNG.hypergeometric(*args, **kwargs)
+
+
+def laplace(*args, **kwargs):
+    return _RNG.laplace(*args, **kwargs)
+
+
+def logistic(*args, **kwargs):
+    return _RNG.logistic(*args, **kwargs)
+
+
+def lognormal(*args, **kwargs):
+    return _RNG.lognormal(*args, **kwargs)
+
+
+def logseries(*args, **kwargs):
+    return _RNG.logseries(*args, **kwargs)
+
+
+def negative_binomial(*args, **kwargs):
+    return _RNG.negative_binomial(*args, **kwargs)
+
+
+def noncentral_chisquare(*args, **kwargs):
+    return _RNG.noncentral_chisquare(*args, **kwargs)
+
+
+def noncentral_f(*args, **kwargs):
+    return _RNG.noncentral_f(*args, **kwargs)
+
+
+def normal(*args, **kwargs):
+    return _RNG.normal(*args, **kwargs)
+
+
+def pareto(*args, **kwargs):
+    return _RNG.pareto(*args, **kwargs)
+
+
+def poisson(*args, **kwargs):
+    return _RNG.poisson(*args, **kwargs)
+
+
+def power(*args, **kwargs):
+    return _RNG.power(*args, **kwargs)
+
+
+def rayleigh(*args, **kwargs):
+    return _RNG.rayleigh(*args, **kwargs)
+
+
+def standard_cauchy(*args, **kwargs):
+    return _RNG.standard_cauchy(*args, **kwargs)
+
+
+def standard_exponential(*args, **kwargs):
+    return _RNG.standard_exponential(*args, **kwargs)
+
+
+def standard_gamma(*args, **kwargs):
+    return _RNG.standard_gamma(*args, **kwargs)
+
+
+def standard_normal(*args, **kwargs):
+    return _RNG.standard_normal(*args, **kwargs)
+
+
+def standard_t(*args, **kwargs):
+    return _RNG.standard_t(*args, **kwargs)
+
+
+def triangular(*args, **kwargs):
+    return _RNG.triangular(*args, **kwargs)
+
+
+def uniform(*args, **kwargs):
+    return _RNG.uniform(*args, **kwargs)
+
+
+def vonmises(*args, **kwargs):
+    return _RNG.vonmises(*args, **kwargs)
+
+
+def wald(*args, **kwargs):
+    return _RNG.wald(*args, **kwargs)
+
+
+def weibull(*args, **kwargs):
+    return _RNG.weibull(*args, **kwargs)
+
+
+def zipf(*args, **kwargs):
+    return _RNG.zipf(*args, **kwargs)
+
+
+beta.__doc__ = _RNG.beta.__doc__
```

Comment:
I think this could be written as a loop like 

```python
for func in [binomial, chisquared, …]:
  func.__doc__ = getattr(_RNG, func.__name__).__doc__
```

Reply:
Yep that's better, fixed in https://github.com/WWakker/mtalg/pull/9/commits/cf3c2246e2a754a78e55c9b965f1f60a9d299278